src/crewai/crew.py

```diff
@@ -235,6 +238,9 @@ def kickoff(self, inputs: Optional[Dict[str, Any]] = {}) -> str:
 
         i18n = I18N(language=self.language, language_file=self.language_file)
 
+        if agentops:
```

Comment:
@joaomdmoura: this addition means that if the developer is using agentops, we will set the parent key. This way, Crew will have access to some run data in this case.

we can also change the conditional to `if agentops and self.share_crew` if you'd rather